ecommerce-ios/Sources/Home/SuggestedView/SuggestedView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  SuggestedView.swift
+//  ecommerce-ios
+//
+//  Created by Su T. Nguyen on 14/05/2021.
+//
+
+import SwiftUI
+
+struct SuggestedView: View {
+
+    private let product = Product(
+        id: "product1",
+        name: "Poly Cube 2021",
+        imageName: "",
+        price: Price(id: "price1", amount: 8_000, currency: "THB")
+    )
+
+    var body: some View {
+        ZStack {
+            backgroundView
+            contentView
+        }
+        .clipped()
+    }
+
+    private var backgroundView: some View {
+        ZStack {
+            Color.darkBlue
+                .ignoresSafeArea(edges: .top)
+                .frame(height: 386.0)
+
+            Image("product-cube")
+                .offset(x: 45.0, y: 45.0)
+        }
+    }
+
+    private var contentView: some View {
+        VStack(alignment: .leading) {
+            headerView
+
+            Spacer()
+                .foregroundColor(.white)
+            Rectangle()
+                .frame(height: 1)
+                .foregroundColor(.white)
+                .opacity(0.2)
+
+            footerView
+        }
+        .padding(.all, 15.0)
```

Comment:
We can shorten `.padding(.all, 15.0)` by:

```suggestion
        .padding(15.0)
```